docs/examples/plot_dataset_tutorial.py

```diff
@@ -0,0 +1,72 @@
+"""
+An Example of Using Data Stored in Different Forms.
+===================================================
+
+Different data sets are stored in various structures and formats.
+To apply LibMultiLabel with any of them, one must convert the data to a form accepted by the library first.  
+In this tutorial, we demonstrate an example of conversion of a hugging face data set.
+Before we start, note that LibMultiLabel format consists of IDs (optional), labels, and raw texts. 
+`Here <https://www.csie.ntu.edu.tw/~cjlin/libmultilabel/cli/ov_data_format.html#libmultilabel-format>`_ are more details. 
+
+To begin, let's install ``datasets`` from the hugging face library by the following command::
+
+  pip3 install datasets
+
+We use Pandas' ``DataFrame`` to process the data set from Hugging Face
+and load the data set by Hugging Face's API ``load_dataset``.
+Please import these libraries. 
+"""
+
+import pandas
+from datasets import load_dataset
+
+######################################################################
+# We choose a multi-label set ``emoji`` from ``tweet_eval`` in this example. 
+# The data set can be loaded by the following code.
+
+data_sets = dict()
+data_sets['train'] = load_dataset('tweet_eval', 'emoji', split='train')
+data_sets['val'] = load_dataset('tweet_eval', 'emoji', split='validation')
+data_sets['test'] = load_dataset('tweet_eval', 'emoji', split='test')
+
+######################################################################
+# Convert to LibMultiLabel format
+# --------------------------------------------------
+# We begin with loading Hugging Face's data set to Pandas' structure by the function ``DataFrame``. 
+# In consistence with our `linear model quickstart <https://www.csie.ntu.edu.tw/~cjlin/libmultilabel/auto_examples/plot_linear_quickstart.html>`_ which don't have validation set,
+# we use ``pandas.concat`` to merge training and validation sets and use ``reset_index`` to add the new indices to rows.
+
+for d_set in ['train', 'val', 'test']:
```

Comment:
How about replacing `d_set` with `split` which is aligned with [BERT's](https://www.csie.ntu.edu.tw/~cjlin/libmultilabel/auto_examples/plot_bert_quickstart.html)?

Reply:
Ok. Done.